packages/xstate-inspect/test/inspect.test.ts

```diff
@@ -340,4 +340,40 @@ describe('@xstate/inspect', () => {
         .filter((message: any) => message.type === 'service.event')
     ).toHaveLength(1);
   });
+
+  describe('browser inspector', () => {
```

Comment:
I would unnest this block - just add the new tests to the existing block. I find `describe` blocks to be hard to work with because people often accidentally insert new tests in a "random" block (well, usually in the last one) without realizing that they are actually in a block.

Reply:
Sounds good. I'll make this change.